rebar.config

```diff
@@ -75,37 +86,28 @@
   {eodbc, {git, "https://github.com/arcusfelis/eodbc.git", {branch, "master"}}},
   {tirerl, "1.1.0"},
 
-  {cache_tab, "1.0.28"},
-  {jid, "1.0.0", {pkg, mongoose_jid}},
-
   {proper, "1.4.0"},
   {meck, "0.8.13"},
```

Comment:
Not this PR, but how is it that we need these two... 🤔

Reply:
Those are funny ones. They could be declared as dependencies in the test profile for rebar, but, big tests are actually rpc-ing to them to do some black magic 🤷🏽‍♂️

Reply:
yea, but they don't go into the release build, so it's fine